x/ecocredit/client/testsuite/tx.go

```diff
@@ -441,7 +517,7 @@ func (s *IntegrationTestSuite) TestTxSendCmd() {
 
 	for _, tc := range testCases {
 		s.Run(tc.name, func() {
-			cmd := coreclient.TxSendCmd()
```

Comment:
Ah. One more thing. We need to keep `TxSendCmd` and add `TxSendBulkCmd` rather than only adding `TxSendBulkCmd`. This will make both commands available to use.